src/skins/vector/css/matrix-react-sdk/views/messages/MTextBody.css

```diff
@@ -13,12 +13,3 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-
-.mx_MTextBody {
-    white-space: pre-wrap;
```

Comment:
Why?  We no longer preserve whitespace in messages now - for instance, trying to type `:    (` is rendered as `: (` rather than with all the spaces intact.